posthog/views.py

```diff
@@ -91,6 +92,8 @@ def security_txt(request):
 
 @never_cache
 def preflight_check(request: HttpRequest) -> JsonResponse:
+    slack_client_id = SlackIntegration.slack_config().get("SLACK_APP_CLIENT_ID")
```

Comment:
Is this all guaranteed to be present? 

Could init to `None` in the `get` instead of on line 112

Reply:
Yeah but the default is empty string so it will always be present, hence the check to force it None if empty.